### PR TITLE
Retoken merge

### PR DESCRIPTION
**Pull request by amyxie361**

### CODE REVIEW

**Comment by amallia** [comment]
Thanks for the PR! I will definitely take a look soon.

File: src/index.py [hunk 1]
```diff
@@ -41,10 +33,12 @@ def tok(d):
 
     return tokenized_content, terms
 
-
+def quantize(value, scale):
+    return int(ceil(value * scale))
 
 def process_batch(g, super_batch):
     print_message("Start process_batch()", "")
+    scale = (1 << 8) / 21.0
```
Comment (amallia) [inline src/index.py:41]: why 21?

File: src/index.py [hunk 2]
```diff
@@ -66,33 +61,67 @@ def process_batch(g, super_batch):
...
-            term_scores = ', '.join([term + ": " + str(round(score, 3)) for term, score in term_scores])
-            lines.append(term_scores)
+        for idx, term_scores in enumerate(every_term_score):
+            data = {
+                    "id":idx,
+                    "contents": "",
+                    "vector":{}
+                    }
+            for t, s in term_scores:
+                data["vector"][t] = quantize(s, scale)
```
Comment (amallia) [inline src/index.py:71]: Why do you want to quantize now?

**Comment by amallia** [comment]
I think it is good to pull from `main` as you are pushing starting from an old checkpoint

File: src/model_multibert.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
+import torch
```
Comment (amallia) [inline src/model_multibert.py:1]: `model_multibert.py` was merged with `model.py` - please pull from the latest commit

**Comment by amallia** [comment]
Could you provide a description of the changes, the improvements if any and the overall idea of this PR? Thanks

**Comment by amyxie361** [comment]
Sorry, Antonio, I messed things up. I intended to merge to my forked repo first. Will try to creat pull request later when I clean everything up.

**Pull request closed without merging**